src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--start-day--wed calendar--month-length--31">
+        <span class="calendar__item calendar__item--01"></span>
```

Comment:
Don't add extra classes here, use `calendar__item` class with `:nth-child`

Reply:
forgot to remove it, done.